### PR TITLE
chore: update the trusted setup to match the consensus-specs version

### PR DESCRIPTION
**Pull request by kevaundray**
## Issue Addressed

This adds the exact trusted file that is in the consensus-specs. The previous(current) file was a truncated version that did not include the g1_monomial points. If these are not included, then the KZG libraries will need to recompute them since peerDAS requires them.

## Proposed Changes

This PR copies the file from the consensus-specs repo and overwrites the previous one. The g1_lagrange and g2_monomial points are the same. Since I wanted to match the checksum in the consensus-specs repository, this does change the formatting. 

## Additional Info

Please provide any additional information. For example, future considerations
or information useful for reviewers.

### CODE REVIEW

**Comment by kevaundray** [comment]
For reference, to check that they are equal -- 

**Compute the checksum for the file in consensus-specs**

```bash
curl -sL https://raw.githubusercontent.com/ethereum/consensus-specs/master/presets/mainnet/trusted_setups/trusted_setup_4096.json | shasum -a 256
```

**Compute the checksum for file in this PR**

```bash
curl -sL https://raw.githubusercontent.com/kevaundray/lighthouse/kw/add-official-trusted-setup/common/eth2_network_config/built_in_network_configs/trusted_setup.json | shasum -a 256
```

Link to consensus-specs: https://github.com/ethereum/consensus-specs/blob/master/presets/mainnet/trusted_setups/trusted_setup_4096.json

**Review by jimmygchen — APPROVED** [review]
Nice, thanks @kevaundray !

> If these are not included, then the KZG libraries will need to recompute them since peerDAS requires them.

Is this the new behaviour for `c-kzg-4844` as well, once the `das` PR (https://github.com/ethereum/c-kzg-4844/pull/452) is merged to main?

Note to other reviewers: `g1_monomial_points` is unused in the code base right now, so this change should have minimal impact.

**Comment by jimmygchen** [comment]
Note: we should deploy this branch to a testnet node before merging.

**Comment by kevaundray** [comment]
> Nice, thanks @kevaundray !
> 
> > If these are not included, then the KZG libraries will need to recompute them since peerDAS requires them.
> 
> Is this the new behaviour for `c-kzg-4844` as well, once the `das` PR ([ethereum/c-kzg-4844#452](https://github.com/ethereum/c-kzg-4844/pull/452)) is merged to main?
> 
> Note to other reviewers: `g1_monomial_points` is unused in the code base right now, so this change should have minimal impact.

Yep this new behaviour is the same in c-kzg-4844. It can be seen in #6119 where I've pulled in the latest commit from the `das` branch; the `KzgSettings::load_trusted_setup` function now requires g1_monomial, g1_lagrange and g2_monomial

**Comment by jimmygchen** [comment]
Looks good on Holesky so far. The two failing tests are likely related to our CI infra and not caused by the changes here.

File: crypto/kzg/src/trusted_setup.rs [hunk 1]
```diff
@@ -21,6 +21,8 @@ struct G2Point([u8; BYTES_PER_G2_POINT]);
 /// See https://github.com/ethereum/consensus-specs/blob/dev/presets/mainnet/trusted_setups/trusted_setup_4096.json
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct TrustedSetup {
+    #[serde(rename = "g1_monomial")]
+    g1_monomial_points: Vec<G1Point>,
     #[serde(rename = "g1_lagrange")]
     g1_points: Vec<G1Point>,
```
Comment (jtraglia) [inline crypto/kzg/src/trusted_setup.rs:27]: For consistency, we should rename `g1_points` to `g1_lagrange_points` and `g2_points` to `g2_monomial_points`.

Comment (jtraglia) [reply crypto/kzg/src/trusted_setup.rs:27]: I see that was done in your other PR, but it should be here IMO.

image

**Review by michaelsproul — APPROVED** [review]

**Comment by jimmygchen** [comment]
@mergify queue

**Pull request merged**